job_templates/ci_job.xml.em

```diff
@@ -180,6 +180,9 @@ echo "# END SECTION"
 echo "# BEGIN SECTION: docker info"
 docker info
 echo "# END SECTION"
+echo "# BEGIN SECTION: Inject date into Dockerfile"
+sed -i "s/@@today_str/`date +%Y-%m-%d`/" linux_docker_resources/Dockerfile
```

Comment:
This is aimed at replacing [this line](https://github.com/ros2/ci/blob/6806ca3f122adef26d9e42d906d7acebc1ff2c4d/linux_docker_resources/Dockerfile#L83) in the Dockerfile at build time.